examples/minimal_hyperslicer.ipynb

```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Minimal Hyperslicer Tutorial\n",
+    "\n",
+    "The hyperslicer function is meant to make it easier to explore imaging hyperstacks. It was designed with hyperspectral Coherent Raman Scattering in mind but should be useful for other advanced microscopy modalities such as FLIM and would play nicely with standard xyzt type stacks.\n",
+    "\n",
+    "Currently this minimal version simply takes an arrary whose last two dimension are the image dimensions and all previous dimensions are taken to represent other slices of the hyperstack. It will automatically generate IntSliders that select the the relevant slice along each dimension of the input array. \n",
+    "\n",
+    "The minimal example here uses a hyperspectral stimulated raman scattering stack of beads that are Raman active at different wavenumbers. You can download the dataset from [here](https://github.com/jrussell25/data-sharing/blob/master/beads_srs_20200818.zip). You can see the two different species with the larger beads (PMMA) peaking around frame 63 and the smaller beads (Polystyrene) peaking around frame 110."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import tifffile as tiff\n",
```

Comment:
if we keep the tiffs as an example then we will need to add `tifffile` to the `test` extrarequies https://github.com/ianhi/mpl-interactions/blob/fd41f510a4c813befa984df76a35e240c382963a/setup.py#L63

part of the tests is that they run every example notebook

Reply:
now have data as npy as use np.load 34b76de8c49d45c9f2f789e12c1b9cfc2d66f90b